docs/site/src/theme/CodeBlock/index.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * SWIZZLED VERSION: 3.5.2
+ * REASONS:
+ *  - Add check for reference CodeBlock so it can coexist with the original CodeBlock and the live code block
+ */
+import React from 'react';
+import CodeBlock from '@theme-original/CodeBlock';
+import ReferenceCodeBlock from '@saucelabs/theme-github-codeblock/build/theme/ReferenceCodeBlock';
+import type CodeBlockType from '@theme/CodeBlock';
+import type {WrapperProps} from '@docusaurus/types';
+
+type Props = WrapperProps<typeof CodeBlockType>;
```

Comment:
We should use proper props here. Not sure how to solve that in a good way. We probably need to extend the original props with both props from the code block plugins 🤔

Reply:
Done. Not sure if this is the proper react way and if I understood everything correctly, but done 😆